Build/ProjectTypes/WebProject.props

```diff
@@ -0,0 +1,14 @@
+<Project>
```

Comment:
We should go with `WebApplication` to distinguish from `WebLibrary`. And we can continue to not make it clear that it's a .NET Core WebApplication since the NET Framework variant is a completely different project-file setup. Plus, there's an expiration date to the ambiguity.